### PR TITLE
Load config from server to be able to change API server URL

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -26,7 +28,7 @@ const METADATA = webpackMerge(commonConfig.metadata, {
   HMR: HMR
 });
 
-const frontendConfig = commonConfig.frontendConfig;
+const frontendConfig = _.merge(commonConfig.frontendConfig, { apiEndpoint: 'http://localhost:8080/v1' });
```
Comment (kahboom) [inline config/webpack.dev.js]: So you did end up deciding that the Webpack configuration was the best way to use env-specific settings? I don't mind moving all globals from Angular to Webpack if you'd prefer. I believe it was set up that way before.

File: config/webpack.common.js [hunk 2]
```diff
@@ -241,17 +243,17 @@ module.exports = {
      */
     new CopyWebpackPlugin([
       {
-        from: {glob: './node_modules/patternfly/dist/img/*.*'},
+        from: { glob: './node_modules/patternfly/dist/img/*.*' },
```
Comment (kahboom) [inline config/webpack.common.js]: Thanks for the formatting 👍

Comment (jimmidyson) [inline config/webpack.common.js]: That's from awesome vscode, not intentional :-b

File: config/webpack.dev.js (hunk 1 above)
Comment (jimmidyson) [inline config/webpack.dev.js]: `Globals` could be used instead I guess, More of a `Defaults` than `Globals` perhaps.

**Comment by jimmidyson** [comment]
Turns out this didn't work at all... Looks like the app initializer stuff is a bit broken here, think we need to separate out provider for app config from app state (which is really a service).

**Pull request merged**